server/config.js

```diff
@@ -1,9 +1,11 @@
 import { join } from 'path'
-import { readFile } from 'mz/fs'
 
 const cache = new Map()
 
-const defaultConfig = { cdn: true }
+const defaultConfig = {
+  cdn: true,
+  webpack: (cfg, hotReload) => cfg
```

Comment:
It's a pretty good idea if we could log something, when we are using this user supplied webpack config decorator.
So, I suggest to not to define this.
And when we are using this, we can check for webpack decorator and log it.